### DISCUSSION TITLE
Use single domain to serve multiple containers on different uri paths

### DISCUSSION THREAD

**Discussion by gbili** [nginx-proxy/nginx-proxy#1860] [Ideas]
Feature description
-----------------

Feature Request. Would it be possible to have multiple containers running under the same domain, but each under their own path.

E.g:
container_a : example.com/a
container_b : example.com/b

**Comment by buchdag** [comment]
Hi. Please see https://github.com/nginx-proxy/nginx-proxy/discussions/1733
